packages/node-opcua-secure-channel/source/security_policy.ts

```diff
@@ -584,7 +588,14 @@ export function verifySignature(
     assert(signature.signature instanceof Buffer);
     // This parameter is calculated by appending the clientNonce to the clientCertificate
     const dataToVerify = Buffer.concat([chain[0], receiverNonce]);
-    return cryptoFactory.asymmetricVerify(dataToVerify, signature.signature, senderCertificate);
+    try {
+        return cryptoFactory.asymmetricVerify(dataToVerify, signature.signature, senderCertificate);
+    } catch (e) {
+        if (doTraceChunk) {
```

Comment:
Let's remove this if statemtn, a warning can be emitted regardless.

Reply:
fixed in commit d413004